[workflow] feat: Fetch settings (#2)
neondatabase/reusable-workflows: README.md

    name: Create/Delete/Reset Branch for Pull Request Job
    uses: neondatabase/reusable-workflows/.github/workflows/neon-preview-branches-for-pull-requests.yml@main
    with:
      # required parameters
      # a Neon project ID
      project_id: proud-salad-35321605
      
      # optional parameters
      # a Neon branch ID to be used as a parent branch for the branches created by the workflow

>       # if not specified, primary branch will be used

Reviewer: ```suggestion
      # if not specified, the primary branch will be used
```

      parent_branch: main

>       # a Neon database name, to a construct DB connection URL after a branch creation

Reviewer: ```suggestion
      # a Neon database name for constructing a database connection URL after a branch creation
```

      db: neondb

>       # a Neon database role, to a construct DB connection URL after a branch creation

Reviewer: ```suggestion
      # a Neon database role for constructing a database connection URL after a branch creation
```

      role: neondb_owner
    secrets:
      NEON_API_KEY: ${{ secrets.NEON_API_KEY }}
```

> If there is a Github integration in the [Neon Console](https://console.neon.tech/) for the project, you may leave only the `project_id`

Reviewer: ```suggestion
If the Neon project has an existing Github integration, you can only specify the `project_id`.
```

> parameter, other parameters will be fetched from the Neon API.

Reviewer: ```suggestion
Other parameters will be fetched via the Neon API.
```
